src/app/public/modules/dropdown/dropdown.component.ts

```diff
@@ -253,6 +258,14 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
       .subscribe((message: SkyDropdownMessage) => {
         this.handleIncomingMessages(message);
       });
+
+    this.themeSvc.settingsChange
+      .pipe(
+        takeUntil(this.ngUnsubscribe)
+      )
+      .subscribe(() => {
+        this.changeDetector.markForCheck();
```

Comment:
I'm guessing this is necessary to get the themeIf's to properly trigger? Would you mind adding a comment here to explain?

Reply:
Added a comment. With OnPush change detection, this is necessary for conditional `<sky-icon>` elements to render when the theme changes.